### DISCUSSION TITLE
Link titles get changed automatically

### DISCUSSION THREAD

**Discussion by robbertdam** [combostrap/combo#76] [Q&A] [answered]
In the frontpage of my wiki I added a link to a 'Getting started guide'

[[guide:Get started]]

But this link renders as follows:

![image](https://github.com/ComboStrap/combo/assets/20517275/d53ef377-14b5-4f03-b803-d137e7e4c93f)

Maybe that's because I initially give the page a header with the text: "Getting started with this wiki".

This seems like an (undocumented) feature. Can I disable this?

**Comment by gerardnico** [comment, marked as answer]
I don't remember the exact rule. I should document it.
By default, a link label in the document body seems to be the title.

For now, you can overwrite the default label with this syntax.
```
[[guide:Get started|Getting started]]
```
